org.apache.jmeter.metainfo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.apache.jmeter</id>
+  <name>Apache JMeter</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  <developer_name>Apache Software Foundation</developer_name>
+  <summary>Load testing and performance measurement application</summary>
+  <description>
+    <p>Apache JMeter is a desktop application to design and run load test and measurements.</p>
+  </description>
+  <url type="homepage">https://jmeter.apache.org/</url>
+  <url type="bugtracker">https://jmeter.apache.org/issues.html</url>
+  <url type="faq">https://cwiki.apache.org/confluence/display/JMETER/JMeterFAQ</url>
+  <url type="help">https://jmeter.apache.org/usermanual/index.html</url>
+  <url type="donation">https://www.apache.org/foundation/contributing.html</url>
+  <url type="contact">http://jmeter.apache.org/mail2.html</url>
+  <!-- TODO
+  <screenshots>
```

Comment:
screenshots are required. the build will fail without

Reply:
Added screenshots from the official README file